File: dotnet/cafdemalihapa/Directories/Repository/DotnetDirectory/Cafdemalihapa/Scripts/EnvironmentVariablesSource/PersistentVariablesFile.cs
namespace cafdemalihapa.Directories.Repository.DotnetDirectory.Cafdemalihapa.Scripts.EnvironmentVariablesSource.Files.Jsons
{
    public class PersistentVariablesFile(
EnvironmentVariablesSourceDirectory environmentVariablesSourceDirectory
    )
    {
        public string GetName()
        {
            return "persistent-variables.json";
        }

        public string GetPath()
        {
            var name = GetName();
            return Path.Combine(environmentVariablesSourceDirectory.GetPath(), name);
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/Scripts/EnvironmentVariablesSource/PersistentVariablesFile.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet.Scripts.EnvironmentVariablesSource.Files.Jsons

open System.IO
open cafdemwimu.console.Directories.Repository.Dotnet.Scripts.EnvironmentVariablesSource

type PersistentVariablesFile(environmentVariablesSourceDirectory: EnvironmentVariablesSourceDirectory) =
    member _.GetName() = "persistent-variables.json"

    member this.GetPath() =
        let name = this.GetName()
        Path.Combine(environmentVariablesSourceDirectory.GetPath(), name)
